examples/magento-graphcms/graphql/ProductPage2.graphql

```diff
@@ -1,11 +1,4 @@
-query ProductPage2(
-  $url: String!
-  $urlKey: String!
-  $reviewPageSize: Int = 3
-  $reviewPage: Int = 1
-) {
-  ...UspsQueryFragment
-  ...PageContentQueryFragment
```

Comment:
With the removal of the UspsQueryFragment and PageContentQueryFragment this query is Magento only and this can be moved to the magento-product package in the future, but leave it here for now.